### ISSUE TITLE
Unable to Provision a new vdb

### ISSUE THREAD

**Issue by pmaluf** [delphix/dxtoolkit#16]
I get the following error when i trying to provision a new vdb using the dx_provision_vdb.pl: 
```
perl dx_provision_vdb.pl -engine delphix-engine -group "MISC QA" -template MISCQA -sourcename MISC -targetname MISCQA2 -dbname MISCQA2 -instname MISCQA2 -uniqname MISCQA2 -environment delphix-db-2-type oracle -envinst "/u00/app/oracle/product/11.2.0/dbhome_1" -debug
Dxtoolkit version 2.3.2-rc1
connecting to: delphix-engine ( IP/name : delphix-engine )
GET: http://delphix-engine:80/resources/json/delphix/session
** GET http://delphix-engine:80/resources/json/delphix/session ==> 200 OK
Session exists.
GET: http://delphix-engine:80/resources/json/delphix/source
** GET http://delphix-engine:80/resources/json/delphix/source ==> 200 OK (2s)
GET: http://delphix-engine:80/resources/json/delphix/sourceconfig
** GET http://delphix-engine:80/resources/json/delphix/sourceconfig ==> 200 OK (1s)
GET: http://delphix-engine:80/resources/json/delphix/repository
** GET http://delphix-engine:80/resources/json/delphix/repository ==> 200 OK
GET: http://delphix-engine:80/resources/json/delphix/environment
** GET http://delphix-engine:80/resources/json/delphix/environment ==> 200 OK
GET: http://delphix-engine:80/resources/json/delphix/environment/oracle/clusternode
** GET http://delphix-engine:80/resources/json/delphix/environment/oracle/clusternode ==> 200 OK
GET: http://delphix-engine:80/resources/json/delphix/environment/windows/clusternode
** GET http://delphix-engine:80/resources/json/delphix/environment/windows/clusternode ==> 200 OK
GET: http://delphix-engine:80/resources/json/delphix/environment/user
** GET http://delphix-engine:80/resources/json/delphix/environment/user ==> 200 OK
GET: http://delphix-engine:80/resources/json/delphix/environment/oracle/listener
** GET http://delphix-engine:80/resources/json/delphix/environment/oracle/listener ==> 200 OK
GET: http://delphix-engine:80/resources/json/delphix/host
** GET http://delphix-engine:80/resources/json/delphix/host ==> 200 OK (1s)
GET: http://delphix-engine:80/resources/json/delphix/namespace
** GET http://delphix-engine:80/resources/json/delphix/namespace ==> 200 OK
GET: http://delphix-engine:80/resources/json/delphix/database
** GET http://delphix-engine:80/resources/json/delphix/database ==> 200 OK (1s)
GET: http://delphix-engine:80/resources/json/delphix/namespace
** GET http://delphix-engine:80/resources/json/delphix/namespace ==> 200 OK
GET: http://delphix-engine:80/resources/json/delphix/group
** GET http://delphix-engine:80/resources/json/delphix/group ==> 200 OK
GET: http://delphix-engine:80/resources/json/delphix/policy
** GET http://delphix-engine:80/resources/json/delphix/policy ==> 200 OK
GET: http://delphix-engine:80/resources/json/delphix/snapshot?database=ORACLE_DB_CONTAINER-173
** GET http://delphix-engine:80/resources/json/delphix/snapshot?database=ORACLE_DB_CONTAINER-173 ==> 200 OK
GET: http://delphix-engine:80/resources/json/delphix/environment
** GET http://delphix-engine:80/resources/json/delphix/environment ==> 200 OK
GET: http://delphix-engine:80/resources/json/delphix/environment/oracle/clusternode
** GET http://delphix-engine:80/resources/json/delphix/environment/oracle/clusternode ==> 200 OK
GET: http://delphix-engine:80/resources/json/delphix/environment/windows/clusternode
** GET http://delphix-engine:80/resources/json/delphix/environment/windows/clusternode ==> 200 OK
GET: http://delphix-engine:80/resources/json/delphix/environment/user
** GET http://delphix-engine:80/resources/json/delphix/environment/user ==> 200 OK
GET: http://delphix-engine:80/resources/json/delphix/environment/oracle/listener
** GET http://delphix-engine:80/resources/json/delphix/environment/oracle/listener ==> 200 OK
GET: http://delphix-engine:80/resources/json/delphix/database/template
** GET http://delphix-engine:80/resources/json/delphix/database/template ==> 200 OK (1s)
GET: http://delphix-engine:80/resources/json/delphix/namespace
** GET http://delphix-engine:80/resources/json/delphix/namespace ==> 200 OK
GET: http://delphix-engine:80/resources/json/delphix/group
** GET http://delphix-engine:80/resources/json/delphix/group ==> 200 OK
GET: http://delphix-engine:80/resources/json/delphix/repository
** GET http://delphix-engine:80/resources/json/delphix/repository ==> 200 OK
	** POST http://delphix-engine:80/resources/json/delphix/database/provision ==> 200 OK
	Problem with starting job
	Error: sourceConfig->instances->This field is required.
	error $VAR1 = {
		  'sourceConfig' => {
				      'instances' => {
						       'commandOutput' => undef,
						       'details' => 'This field is required.',
						       'action' => undef,
						       'id' => 'exception.validation.field.missing',
						       'diagnoses' => undef
						     }
                            }
        };
$VAR2 = 1;

Job wasn't defined. If there is no error on the screen, try with -debug flag to find a root cause.
```

Delphix Engine Version: 5.1.3.1
Build Date: Sat, 5 Nov 2016 00:33:52 GMT

Thanks,
Paulo.

**Comment by pioro** [comment]
Hello, 

Sorry for delay in answer.
Similar issue was identified when RAC to non-RAC provisioning have been used.
Could you please confirm that this is our case as well ?

regards,
Marcin

**Comment by pmaluf** [comment]
Hello Marcin, 
Yes, my dsource is RAC and i tried to provisioning to a non-RAC environment. 

Regards,
Paullo

**Comment by pioro** [comment]
Hi,

It's fixed now in 2.3.2-rc3.
You can try it.

regards,
Marcin

**Comment by pmaluf** [comment]
Hi Marcin, 

The  2.3.2-rc3 works like a charm. 

Starting provisioning job - JOB-7842
0 - 6 - 9 - 10 - 45 - 46 - 47 - 48 - 51 - 60 - 100
Job JOB-7842 finished with state: COMPLETED
VDB created..

Thanks!
